annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/jsonld/BaseJsonldRest.java

```diff
@@ -413,8 +413,8 @@ private Annotation verifyOwnerOrAdmin(String identifier, Authentication authenti
 	    return annotation;
 	}else {
 	    //not authorized
-	    throw new ApplicationAuthenticationException(I18nConstants.OPERATION_NOT_AUTHORIZED,
-		    I18nConstants.OPERATION_NOT_AUTHORIZED, new String[] { "Only the creators of the annotation or admins are authorized to perform this operation."});
+	    throw new ApplicationAuthenticationException("Only the creators of the annotation or admins are authorized to perform this operation.",
```

Comment:
Please revert back this change, otherwise the serialization of the response using the messages_en.properties file will be broken. Please set the correct 403 HttpStatus to the ApplicationAuthenticationException at this stage

Reply:
Done.